### PR TITLE
Feat/keyboard navigation

### PR DESCRIPTION
**Pull request by MartnLecam**
# Name of the feature/issue
Keyboard navigation

## Description
Enable shortcuts on the extension.

### CODE REVIEW

File: packages/ui/src/components/gas/GasPricesInfo.tsx [hunk 1]
```diff
@@ -119,6 +120,8 @@ const GasPricesInfo: FC = () => {
         localeInfo,
         networkNativeCurrency,
         isNetworkChanging,
+        isRatesChangingAfterNetworkChange,
```
Comment (antonyayoub) [inline packages/ui/src/components/gas/GasPricesInfo.tsx:123]: Is isRatesChangingAfterNetworkChange needed here?

Comment (MartnLecam) [reply packages/ui/src/components/gas/GasPricesInfo.tsx]: It is not. Removed! 😁

**Review by julianariel — COMMENTED** [review]
tested it using Windows and it works perfectly 👏🏻 👏🏻  nice UX improvement, I love it!


Added some small comments and visual feedback:


- ALT borders seem different
- Click here should be a `ClickableText` component
image

- this modal shouldn't occupy the whole screen, it should have vertical spacing like every other modal
image

- this modal should have the same spacing between the texts as other confirmation modals
image
image

- modals headers are different from the designs and other modals
image
image

- allowances screen does not have shortcuts, and ALT+K doesn't work. please let's check all the pages.
image

File: public/manifest.json [hunk 2]
```diff
@@ -1,20 +1,12 @@
 {
     "author": "BlockWallet",
     "background": {
-        "scripts": [
```
Comment (julianariel) [inline public/manifest.json:4]: this lint change will cause the pipeline to fail. Please run `yarn run lint:fix` standing in the UI folder to fix it

**Pull request merged**